sched/clustermembership/dynamicCluster.go

```diff
@@ -1,49 +1,77 @@
 package cluster_membership
 
-/*
- * Represents a Cluster of nodes that can change over time.
- * Keeps a Map of the nodeId to Node for fast lookup of nodes.
- */
+//
+// Represents a Cluster of nodes that can change over time.
+// Keeps a Map of the nodeId to Node for fast lookup of nodes.
+//
 type dynamicCluster struct {
 	memberSet   map[string]bool
 	membersList []Node
+	updateCh    chan NodeUpdate
```

Comment:
Oh yes I understand, the idea is that there should only be one watcher per dynamic cluster.  I can update and add a comment.  Will combine the current state with updates as a return.  I agree unless the caller knows the implementation could result in missed updates